projects/typescript-examples/contracts/ControlFlow/contract.e2e.spec.ts

```diff
@@ -0,0 +1,130 @@
+import { Config } from '@algorandfoundation/algokit-utils'
+import { registerDebugEventHandlers } from '@algorandfoundation/algokit-utils-debug'
+import { algorandFixture } from '@algorandfoundation/algokit-utils/testing'
+import { Address } from 'algosdk'
```

Comment:
We should minimize usage of algosdk. Instead of using the Address type, could you use the utils Account type and get the `addr` String?